ui/src/views/Scopes/ScopesetExpander/index.jsx

```diff
@@ -41,8 +42,26 @@ export default class ScopesetExpander extends Component {
     scopeText: '',
   };
 
+  componentDidMount() {
+    const query = parse(this.props.location.search.slice(1));
+    const { scopes } = query;
+
+    if (scopes) {
+      this.setState(() => ({
+        scopeText: scopes.join('\n'),
+      }));
+    }
+  }
```

Comment:
Same thing as previous comment about `componentDidMount`.

Reply:
I have removed the `componentDidMount()` and replaced it with a `constructor` here as well.